.github/workflows/development-environment.yml

```diff
@@ -0,0 +1,97 @@
+name: Development Environment
+on:
+  push:
+    paths:
+    - 'Brewfile'
+    - 'Makefile'
+    - 'setup.py'  # For changes in the sentry CLI used in make bootstrap
+    - 'requirements-*.txt'
+    - 'scripts/*'
+    - 'src/sentry/*'  # Imported in setup.py
```

Comment:
I will remove it for now. The only reason I added is because of the code related to calling the sentry CLI.  Let me know if you want me to spend more time to determine what exact modules represent the CLI in order to add it here.

For now, it's fine to drop it.

Reply:
We can start with `devserver.py`, `devservices.py`?